Update Getting Started section [8348] (#112)
eProsima/Fast-RTPS-docs: docs/fastdds/getting_started/simple_app/includes/publisher.rst

2.  Assigns a name to the participant through the QoS of the DomainParticipant.
3.  Uses the DomainParticipantFactory to create the participant.
4.  Registers the data type defined in the IDL.
5.  Creates the topic for the publications.
6.  Creates the publisher.
7.  Creates the DataWriter with the listener previously created.

As you can see, the QoS configuration for all entities, except for the participant's name, is the default configuration
(``PARTICIPANT_QOS_DEFAULT``, ``PUBLISHER_QOS_DEFAULT``, ``TOPIC_QOS_DEFAULT``, ``DATAWRITER_QOS_DEFAULT``).
The default value of the QoS of each DDS Entity can be checked in the
`DDS standard <https://www.omg.org/spec/DDS/About-DDS/>`_.

.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
    :language: C++
    :lines: 113-155

To make the publication, the public member function ``publish()`` is implemented.
In the data writer's listener callback which states that the data writer has matched with a data reader
that listens to the publication topic, the data member ``matched_`` is updated. It contains the number of data readers
discovered.

> Therefore, when the first data reader has been discovered, it starts to publish.

Reviewer: ```suggestion
Therefore, when the first data reader has been discovered, the application starts to publish.
```

> This is actually a `write` of a change of the DataWriter object.

Reviewer: ```suggestion
This is simply the `writing` of a change by the DataWriter object.
```

.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
    :language: C++
    :lines: 157-167

> The public run function executes the action of publishing a given number of times.

Reviewer: ```suggestion
The public run function executes the action of publishing a given number of times, waiting for 1 second between publications.
```
